include/core/CStreamWriter.h

```diff
@@ -0,0 +1,44 @@
+/*
+* Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+* or more contributor license agreements. Licensed under the Elastic License
+* 2.0 and the following additional limitation. Functionality enabled by the
+* files subject to the Elastic License 2.0 may only be used in production when
+* invoked by an Elasticsearch process with a license key installed that permits
+* use of machine learning features. You may not use this file except in
+* compliance with the Elastic License 2.0 and the foregoing additional
+* limitation.
+*/
+
+#ifndef ML_CSTREAMWRITER_H
```

Comment:
I think it's best to stick to a pattern across all files:

```suggestion
#ifndef INCLUDED_ml_core_CStreamWriter_h
```